websites/W/Warriorcats/presence.ts

```diff
@@ -0,0 +1,270 @@
+const presence = new Presence({
+		clientId: "936630095035113522",
+	}),
+	browsingTimestamp = Math.floor(Date.now() / 1000);
+
+async function getPostNumber() {
+	// find <article></article> and get the #post- number
+	let postNumber = document.querySelector("article").getAttribute("id");
+	postNumber = postNumber.replace(/post-/, "");
+	return postNumber;
+}
+
+presence.on("UpdateData", async () => {
+	const presenceData: PresenceData = {
+		details: "Viewing an unsupported page",
+		state: "",
+		largeImageKey:
+			"https://upload.wikimedia.org/wikipedia/commons/thumb/c/cf/Warrior_Cats_logo.svg/2560px-Warrior_Cats_logo.svg.png",
+		startTimestamp: browsingTimestamp,
+		buttons: [
+			{ label: "Current Page", url: "https://www.warriorcats.com/" },
+			{ label: "Warrior Cats Germany", url: "https://www.warriorcats.de/" },
```

Comment:
[Buttons](https://github.com/PreMiD/Presences/blob/main/.github/CONTRIBUTING.md#presencets) that lead to a home page are now allowed.